includes/Modules/Analytics/Measurement_Events/Measurement_Event.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Measurement_Events\Measurement_Event
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+// phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound
+
+namespace Google\Site_Kit\Modules\Analytics\Measurement_Events;
+
+/**
+ * Use to instantiate MeasurementEvent objects
+ *
+ * @class Measurement_Event_Builder
+ */
+class Measurement_Event_Builder {
```

Comment:
While the builder approach can generally be useful, I think it's a bit overkill here. I think we should simplify the code to get rid of `Measurement_Event_Builder` and instead pass the `array $config` directly to `Measurement_Event` instances.

(Also, there are two classes in this file which should not be the case, but anyway I think we should remove the builder one.)